pom.xml

```diff
@@ -99,14 +99,15 @@
 				<scope>provided</scope>
 			</dependency>
 			<dependency>
-				<groupId>javax.servlet</groupId>
-				<artifactId>jsp-api</artifactId>
-				<version>2.0</version>
+			    <groupId>javax.servlet.jsp</groupId>
+			    <artifactId>javax.servlet.jsp-api</artifactId>
+			    <version>2.3.3</version>
+			    <scope>provided</scope>
```

Comment:
Is adding the scope as provided, part of the ticket?

Reply:
@dkayiwa  it is not part of the ticket but I thought they move together, as it appears in the central maven repo.  Do I have to remove  the scope?

Reply:
Hello @dkayiwa  after removing scope, it worked like magic, the build is passing now. Thank you so much.